client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js

```diff
@@ -20,14 +20,26 @@ import {
 } from '@automattic/wpcom-checkout';
 
 /**
+/**const
  * Internal dependencies
  */
-import { isPlan, isMonthly, getYearlyPlanByMonthly, getPlan } from '@automattic/calypso-products';
-import { isJetpackSite } from 'calypso/state/sites/selectors';
+import {
+	isPlan,
+	isMonthly,
+	getYearlyPlanByMonthly,
+	getPlan,
+	isFreePlanProduct,
+} from '@automattic/calypso-products';
```

Comment:
Minor, but technically `@automattic/calypso-products` is an external dependency (this is left-over from when these imports were automatically converted so nothing to do with you).